### DISCUSSION TITLE
Configuration export for backup / gitops purposes

### DISCUSSION THREAD

**Discussion by tata2000** [airbytehq/airbyte#31177] [Connector Questions]
Hi all!

I'm trying to understand what is the prefferable way to export/save the configuration of my local airbyte installation (not cloud).

I plan on using it for the following :
1. backup before system upgrade.
2. backup and restore of servers in case of failure
3. creation of new environments
4. propagatioin of changes between environments.

The upgrade procedure suggest that i make export via ui for previous to 0.27.0-alpha version, but this is not available anymore.   So how can one upgrade it's installation without loosing all of the configurations?

I have found [that](https://discuss.airbyte.io/t/not-able-to-find-export-import-configuration-option/3487 ) post suggesting that Octavia-cli is the way to go, but now [in the cli documentation]( https://docs.airbyte.com/cli-documentation) it says:
> CAUTION
> The Octavia CLI is an alpha, unofficial CLI that won't be maintained.

But i haven't find any deprecated notation on the readme page of octavia-cli.

I'm confused. What is the best way to handle this? Is there a suggested tool for performing the export? 
Many thanks in advance for all the answers and directions.

**Comment by mcarriere** [comment]
I'm in the same boat and now Octavia cli seems to be gone from Github too. What are the options to be able to export configurations now?

[a comment by an automated account is not shown]
